config/sidekiq-heroku.yml

```diff
@@ -1,5 +1,6 @@
 :concurrency: 2
 :queues:
   - default
+  - mailers
```

Comment:
We could instead just set the mailer queue to `default`. I don't think we gain much by maintaining a separate queue.

```ruby
# config/environments/production.rb
config.action_mailer.deliver_later_queue_name = 'default_mailer_queue'
```

Reply:
@harimohanraj89 I think that's fine. Done https://github.com/simpledotorg/simple-server/pull/1947/commits/ef7b4bbbe8259559d83ee4a394f7957839017991.

This also puts proper priority for `default` because otherwise it's [order](https://github.com/mperham/sidekiq/wiki/Advanced-Options#queues) dependent.

Reply:
Lol good one.